front/src/modules/trainschedule/components/ManageTrainSchedule/ManageTrainScheduleMap/setPointIti.ts

```diff
@@ -1,19 +1,19 @@
 /* eslint-disable import/prefer-default-export */
 import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
 import { insertViaFromMap } from 'reducers/osrdconf/helpers';
-import type { ConfSliceActions } from 'reducers/osrdconf/osrdConfCommon';
 import type { PathStep } from 'reducers/osrdconf/types';
 import { store } from 'store';
 import { addElementAtIndex, replaceElementAtIndex } from 'utils/array';
 
+import type { FeatureInfoClick } from '../types';
+
 export function setPointIti(
   pointType: 'origin' | 'destination' | 'via',
   pathStep: PathStep,
-  actions: ConfSliceActions,
   launchPathfinding: (newPathSteps: (PathStep | null)[]) => void,
+  setFeatureInfoClick: React.Dispatch<React.SetStateAction<FeatureInfoClick | undefined>>,
```

Comment:
It should be resetFeatureInfoClick here
```suggestion
  resetFeatureInfoClick: () => void,
```

Reply:
Done